### PR TITLE
Cut 0.11.0

### PR DESCRIPTION
**Pull request by Nexucis**
to be merged after #91

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
+0.11.0 / 2020-12-08
+===================
+
+* **[Feature]**: Add the completion of the keyword `bool` (#89)
+* **[Feature]**: Add a function `enricher` that can be used to enrich the completion with a custom one
```
Comment (juliusv) [inline CHANGELOG.md:5]: Is this a duplicate of the point below ("Allow the possibility to inject a custom `CompleteStrategy`")?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+* **[Feature]**: Add a LRU caching system.(#71)
+* **[Feature]**: you can now configure the maximum number of metrics in Prometheus for which metadata is fetched.
+* **[Feature]**: Allow the possibility to inject a custom `CompleteStrategy` (#83)
+* **[Feature]**: Provide the Matchers in the PrometheusClient for the method `labelValues` and `series` (#84)
+* **[Feature]**: Add the method `metricName` in the PrometheusClient that supports a prefix of the metric searched. (#84)
+* **[Enhancement]**: Caching mechanism and PrometheusClient are splitted. (#71)
+* **[Enhancement]**: Optimize the code of the PrometheusClient when no cache is using
```
Comment (juliusv) [inline CHANGELOG.md:12]:
```suggestion
* **[Enhancement]**: Optimize the code of the PrometheusClient when no cache is being used.
```

**Comment by juliusv** [comment]
:+1: besides comments

File: CHANGELOG.md (hunk 1 above)
Comment (Nexucis) [reply CHANGELOG.md:5]: ah no it's two different things. `enricher` won't override the default completion strategy, but you can implement your own `CompletionStrategy` and do not use the default one if you want

**Pull request merged**